### DISCUSSION TITLE
nvim-spectre repository URL

### DISCUSSION THREAD

**Discussion by patrick-5546** [LazyVim/LazyVim#665] [Q&A] [answered]
The nvim-spectre URL that LazyVim uses is https://github.com/windwp/nvim-spectre, but has this repository has been moved to https://github.com/nvim-pack/nvim-spectre? The first URL now redirects to the second URL. However, I get the following error when trying to add a keymap:

```
lazy.nvim Two plugins with the same name and different url
```

**Comment by folke** [comment, marked as answer]
I just pushed a fix that automatically handles repo renames. Will create a new release in a bit

**Reply by patrick-5546** [reply]
Thanks!

**Reply by folke** [reply]
new version is now live
